src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,39 @@
+.catalog {
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(1, $width);
+  gap: 48px;
+
+  // display: flex;
+  // flex-wrap: wrap;
+
+  // max-width: 944px;
+  margin: 0 auto;
+
+  &__card {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 200px;
+    height: 300px;
+    border: 1px solid #ccc;
+    border-radius: 5px;
+    margin-bottom: 48px;
+  }
+
+  &__card:not(:nth-child(4n)) {
+    margin-right: 48px;
+  }
```

Comment:
remove this 
![image](https://user-images.githubusercontent.com/91411692/227256669-3e210966-0f72-4d0e-8522-742f80b91538.png)

```suggestion
  }
```